usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Meal.java

```diff
@@ -0,0 +1,65 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public class Meal {
+    private Date date;
+    private MealData body;
+
+    public Meal(Date date, MealData body) {
+        this.date = date;
+        this.body = body;
+    }
+
+    /**
+     * Returns the meal date
+     * @return Meal date
+     */
+    public Date getDate() {
+        return date;
+    }
+
+    /**
+     * Sets a new meal date
+     * @param date Meal date
+     */
+    public void setDate(Date date) {
+        this.date = date;
+    }
+
+    /**
+     * Returns the Meal Data
+     * @return Meal Data
+     */
+    public MealData getBody() {
+        return body;
+    }
+
+    /**
+     * Sets a new Meal Data
+     * @param body Meal Data
+     */
+    public void setBody(MealData body) {
+        this.body = body;
+    }
+
+
+    @NonNull
+    @Override
+    public String toString() {
+        return "{"
+            + "name='" + date.toString() + '\''
+            + ", body=" + body
+            + '}';
+    }
+
+    @Override
+    public boolean equals(@Nullable Object obj) {
+        if (obj instanceof Meal) {
+            return ((Meal) obj).getDate().equals(this.getDate())
+                && ((Meal) obj).getBody()==(this.getBody());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['==' is not preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5314325)